src/monattack.cpp

```diff
@@ -260,6 +263,74 @@ bool mattack::antqueen(monster *z)
     return true;
 }
 
+bool mattack::acidantqueen(monster *z)
```

Comment:
This looks like a copy+paste of regular ant queen function. Is it?
You should extract a common function out of those two. Note that the functions mostly differ in monster types. You could easily extract `mon_ant_acid_larva` and `mon_ant_acid` into `larva_type` and `worker_type` arguments.

Reply:
Again, not sure what you're asking for? I essentially copied and pasted the antqueen special attack and adjusted the necessary ids to reflect the new acid ant queen as well as the larva/worker.

If you're suggesting some further coding development, that's much beyond my capabilities.

Reply:
Copying a whole function and changing a few variables is not the right way to change a function.
Instead of doing it like:
```
void function1() {
    do_stuff(a);
    do_stuff(b);
    do_stuff(c);
    do_stuff(d1);
}
void function2() {
    do_stuff(a);
    do_stuff(b);
    do_stuff(c);
    do_stuff(d2);
}
```
You should do it like:
```
void common_function(argument_type arg) {
    do_stuff(a);
    do_stuff(b);
    do_stuff(c);
    do_stuff(arg);
}
void function1() {
    common_function(d1);
}
void function2() {
    common_function(d2);
}
```

Reply:
Ok, that's what I figured you were getting at. Well, the first example is my limitation of my "coding"/programming ability. The second is far beyond that.